### PR TITLE
Airbrake init

### PR DESCRIPTION
**Pull request by sviik**
A Node.js module for creating node-airbrake and winston-airbrake instances.

Takes a hash (containing configuration options) for an argument and returns an airbrake or winston-airbrake instance.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/salemove/node-airbrake-init.git"
+  },
+  "keywords": [
+    "airbrake", "winston-airbrake"
+  ],
+  "author": "Siim Viiklaid <siim.viiklaid@salemove.com>",
```
Comment (willmore) [inline package.json]: Change to "SaleMove TechMovers <techmovers@salemove.com>"

File: package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "license": "Apache-2.0",
```
Comment (willmore) [inline package.json]: @paniagua do we have any guidelines as to what license to use for our OS stuff? Trivial here, but good to know.

File: test/airbrake_init_test.coffee [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    'whiteListKeys': ['FOO']
+    'blackListKeys': ['BAR']
+    'ignoredExceptions': ['plzignore']
+    'env': 'prod'
+
+  filtered = '[FILTERED]'
+
+  describe 'vanilla airbrake', ->
+
+    createAirbrakeWithConfigEntry = (key, value) ->
```
Comment (willmore) [inline test/airbrake_init_test.coffee]: My preference is to push helper methods like these to the bottom of the file - let the important stuff (tests) be the first thing you see. Just a suggestions.

**Comment by willmore** [comment]
In future, please give commit messages more meaning titles than "Minor fixes". If it is a fixup commit, simply use that mechanism.

**Comment by willmore** [comment]
+1

File: lib/airbrake_init.coffee [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  checkRequiredFields(opts, isWinstonAirbrake: false)
+
+  airbrake = require('airbrake').createClient(opts.projectId, opts.apiKey)
+
+  airbrake.env = opts.env if opts.env
+  airbrake.protocol = opts.protocol if opts.protocol
+  airbrake.servicehost = opts.host if opts.host
+  airbrake.developmentEnvironments = opts.developmentEnvironments if opts.developmentEnvironments
+  airbrake.blackListKeys = opts.blackListKeys if opts.blackListKeys
+  airbrake.whiteListKeys = opts.whiteListKeys if opts.whiteListKeys
```
Comment (stenver) [inline lib/airbrake_init.coffee]: This always exists

Comment (willmore) [inline lib/airbrake_init.coffee]: Yup - blackListKeys, whiteListKeys, ignoredExceptions guaranteed to be present, the above aren't.
I say in this case, while redundant, it is stylistically consistent with the rest of the method.

**Comment by stenver** [comment]
!squash

**Pull request merged**